lib/health/stats/log.ex

```diff
@@ -16,6 +16,6 @@ defmodule Health.Stats.Log do
     log
     |> cast(attrs, [:weight, :date, :user_id])
     |> validate_required([:weight, :date, :user_id])
-    |> validate_number(:weight, greater_than: 100, less_than: 500)
+    |> validate_number(:weight, greater_than: 1, less_than: 500)
```

Comment:
100? really?

Reply:
I’m worried about anorexics using this code! And midgets!

If someone weighs 1 lb the have no business using this app!